### PR TITLE
feat: added score in UI

### PR DESCRIPTION
**Pull request by 0xPratik**

### CODE REVIEW

File: src/view/com/rewards/ScoreCard.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { usePalette } from "../../../lib/hooks/usePalette";
+import { useStores } from "../../../state";
+
+type ScoreCardProps = {
+  style?: StyleProp<ViewStyle>;
+};
+
+export const ScoreCard = observer(({ style }: ScoreCardProps) => {
+  const store = useStores();
+  const did = store.session?.currentSession?.did ?? "";
```
Comment (tj-dispatch) [inline src/view/com/rewards/ScoreCard.tsx:17]: We'll want these 2 lines in the store.

score(did: string) {
 return this.blah blan
}

Comment (tj-dispatch) [reply src/view/com/rewards/ScoreCard.tsx:17]: also, we'll want this on each profile page I think. so we'll want some kind of loading state.

**Review by tj-dispatch — APPROVED** [review]
looks good! just move the score into the store and i'll see what I can do about the other profiles.

do me a favor and leave a comment here with a link to where you'd add this on each profile

File: src/view/com/rewards/ScoreCard.tsx (hunk 1 above)
Comment (0xPratik) [reply src/view/com/rewards/ScoreCard.tsx:17]: got it making the changes

Comment (0xPratik) [reply src/view/com/rewards/ScoreCard.tsx:17]: on Desktop it's on the left Nav and for Mobile it's in the drawer. We could possibly add the score to Profile page as well will discuss this with Robert.

**Pull request merged**